VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -540,8 +540,10 @@ def get_wala_version_from_command(self):
             while(process_wait_time > 0 and p.poll() is None):
                 time.sleep(1)
                 process_wait_time -= 1
-            out = p.stdout.read()
+            out = p.stdout.read().decode()
             out = str(out)
+            if len(out) == 0:
+                raise Exception("/usr/sbin/waagent returns null")
```

Comment:
does raising this exception cause backup failure. If Yes, don't throw any exception. this can be a soft failure.

Reply:
I added it for testing, by mistake it came in the PR.removing it.